frontend/scripts/buildCache.ts

```diff
@@ -51,36 +51,60 @@ async function testAPIAccess() {
   }
 }
 
-function cacheRPCensus() {
-  return seq(...DATASETS.map(_ => async () => {
-    console.log('  Getting RPs and Census for ' + safeDatasetHint(_))
-    let representativePoints = await getRepresentativePoints({ service_area_ids: _.serviceAreaIds })
-    let census = await getCensusData({ service_area_ids: _.serviceAreaIds })
+function cacheData() {
+  return seq(...DATASETS.map(dataset => async () => {
+    console.log('  Getting Rps and Census for ' + safeDatasetHint(dataset))
+    let representativePoints = await getRepresentativePoints({ service_area_ids: dataset.serviceAreaIds })
+    let census = await getCensusData({ service_area_ids: dataset.serviceAreaIds })
+    let storeLikeRps = representativePoints.map(_ => ({
+      ..._,
+      population: Number(_.population),
+      serviceAreaId: _.service_area_id,
+      demographics: census[_.service_area_id]
+    }))
+
     if (uploadToS3) {
-      let rpParams = { Bucket: s3Bucket, Key: getS3Key(getStaticRPUrl(_)), Body: JSON.stringify(representativePoints), ContentType: 'application/json', ACL: 'public-read' }
```

Comment:
here, a log would be helpful to say "Starting upload of website cache to S3 to bucket [blah]"